foundation/foundation-mda/src/main/resources/templates/data-delivery-pyspark/persist.py.vm

```diff
@@ -8,32 +8,26 @@
    #end
   
 
-   #if ($step.isPersistTypeRdbms())
-   #set ($db_url = "db_url: str")
-    def get_connection(self, ${db_url}):
-      """
-      :param db_url: a string that provides the necessary connection details to establish a connection with the RDBMS server.
-      For example, depending on the databse you choose:
-          PostgreSQL: 'postgresql://username:password@host:port/database'
-          MySQL: 'mysql://username:password@host:port/database'
-          SQLite: 'sqlite:///db.sqlite'
-      :return: A connection object from the RDBMS connection pool.
-      """
-      connection = self.rdbms_connection_pool.get_connection(db_url)
-      return connection
-    #end
-
```

Comment:
I: Pruning the use of SQLAlchemy - cannot be used with jdbc protocol to drive spark changes.